ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource

```diff
@@ -53,3 +57,53 @@ Convert Go Test Results To Junit
     IF    ${result.rc} != 0
         FAIL    Failed to convert Go test results to Junit
     END
+
+Run Codeflare Upgrade Tests
```

Comment:
Can this code be adjusted to run any SDK test?
That way we can avoid code duplication from https://github.com/red-hat-data-services/ods-ci/blob/master/ods_ci/tests/Tests/650__distributed_workloads/test-run-codeflare-sdk-e2e-tests.robot and reference there this function.

Though this can be addressed separately.

Reply:
Yes we can adjust it having multiple parameter with Test Name and test type (E2E and upgrade)

Reply:
Created a Jira for this https://issues.redhat.com/browse/RHOAIENG-8564